src/libpriv/rpmostree-passwd-util.c

```diff
@@ -213,6 +228,152 @@ compare_group_ents (gconstpointer a, gconstpointer b)
   return strcmp ((*sa)->name, (*sb)->name);
 }
 
+gboolean
+rpmostree_passwd_ents2sysusers (gboolean is_passwd,
```

Comment:
The wrapper function here feels unnecessary, let's just inline it into the one caller?